src/libvirtApi/network.js

```diff
@@ -62,64 +62,61 @@ export function networkDeactivate({ connectionName, objPath }) {
     return call(connectionName, objPath, 'org.libvirt.Network', 'Destroy', [], { timeout, type: '' });
 }
 
-export function networkGetAll({
+export async function networkGetAll({
     connectionName,
 }) {
-    return call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'ListNetworks', [0], { timeout, type: 'u' })
-            .then(objPaths => {
-                return Promise.all(objPaths[0].map((path) => networkGet({ connectionName, id: path })));
-            })
-            .catch(ex => {
-                console.warn('GET_ALL_NETWORKS action failed:', ex.toString());
-                return Promise.reject(ex);
-            });
+    try {
+        const [objPaths] = await call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'ListNetworks', [0], { timeout, type: 'u' });
+        await Promise.all(objPaths.map((path) => networkGet({ connectionName, id: path })));
+    } catch (ex) {
+        console.warn('GET_ALL_NETWORKS action failed:', ex.toString());
+        throw ex;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1280-20240125-124532-d8538f72-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)